app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert split_integer(10, 5) == [2, 2, 2, 2, 2]
```

Comment:
The test is named: test_sum_of_the_parts_should_be_equal_to_value
But you just test result